packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -5657,12 +5666,25 @@ describe('plugin-meetings', () => {
           assert.equal(meeting.meetingJoinUrl, expectedInfoToParse.meetingJoinUrl);
           assert.equal(meeting.owner, expectedInfoToParse.owner);
           assert.equal(meeting.permissionToken, expectedInfoToParse.permissionToken);
+          assert.equal(meeting.permissionTokenPayload, expectedInfoToParse.permissionTokenPayload);
           assert.deepEqual(meeting.selfUserPolicies, expectedInfoToParse.selfUserPolicies);
         };
 
         it('should parse meeting info from api return when locus meeting object is not available, set values, and return null', () => {
           meeting.config.experimental = {enableMediaNegotiatedEvent: true};
           meeting.config.experimental.enableUnifiedMeetings = true;
+
+          const permissionTokenPayloadData = {
+            exp: '12345678',
+            permission: {
+              userPolicies: {
+                a: true
+              }
+            }
+          };
+
+          const jwtDecodeStub = sinon.stub(jwt, 'decode').returns(permissionTokenPayloadData);
```

Comment:
hmm not sure about the changes in this test. Why do we need to stub jwt.decode now? the permissionToken in FAKE_MEETING_INFO seems to be already populated as a JWT with the data that test expects, can't we just let the decode actually run and decode it?